src/VR.js

```diff
@@ -306,7 +308,7 @@ class FakeVRDisplay extends VRDisplay {
   constructor(window) {
     super('FAKE');
 
-    this.window = window;
+    this.window = globalWindow = window;
```

Comment:
I don't think `globalWindow` passed here is a valid approach; `Gamepad`s can be created from many different contexts, perhaps with no `window` global at all.

If the intent is to bind it for the sake of `getGamepads`, we can pass that in as a closure on `getGamepads` from wherever `navigator` is created (`Window.js`).